installing/installing_aws/preparing-to-install-on-aws.adoc

```diff
@@ -0,0 +1,49 @@
+[id="preparing-to-install-on-aws"]
+= Preparing to install on AWS
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-aws
+
+toc::[]
+
+[id="preparing-to-install-on-aws-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-aws"]
+== Requirements for installing {product-title} on AWS
+
+Before installing {product-title} on Amazon Web Services (AWS), you must create an AWS account. See xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configuring an AWS account] for details about configuring an account, account limits, account permissions, IAM user setup, and supported AWS regions.
+
+See xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[Manually creating IAM for AWS] for the options that are available if the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace.
+
+[id="choosing-an-method-to-install-ocp-on-aws"]
+== Choosing a method to install {product-title} on AWS
+
+You can install {product-title} on AWS by using one of the following deployment methods.
```

Comment:
Suggestions for this list:
- Most unordered lists with item descriptions in the docs seem to have the form of making the term or phrase bold, followed by a colon, and the text after ([example](https://docs.openshift.com/container-platform/4.6/welcome/index.html#developer-activities))
- In this case, it might also help to put the item description (i.e. the text that isn't a link) in a line below the bullet, since some of the items are sort of long phrases. 
- It would be good to see if the description text for each could be somewhat more parallel. Right now, sometimes the description talks about "you can...", or "this method...", or "by using...", etc. It's not too jarring in this case, but might read a bit cleaner that way :smile:

Reply:
This is really good feedback, thank you!

I have shortened the longer title phrases at the start of the bullets so that they are all of a similar length. That way we don't have to drop the descriptions to the subsequent lines. I have also made the bullet titles bold and replaced the full stops with colons, for consistency with the example that you provided.

I have also introduced more parallelism into the list now.